packages/streaming-image-volume-loader/src/BaseStreamingImageVolume.ts

```diff
@@ -731,6 +721,38 @@ export default class BaseStreamingImageVolume extends ImageVolume {
     });
   }
 
+  private _prefetchImageIds() {
+    // Note: here is the correct location to set the loading flag
+    // since getImageIdsRequest is just grabbing and building requests
+    // and not actually executing them
+    this.loadStatus.loading = true;
+
+    const imageIds = this.getImageIdsLoad();
+    this.numFrames = imageIds.length;
+
+    this.totalNumFrames = this.imageIds.length;
+    const autoRenderPercentage = 2;
```

Comment:
shouldn't we use from this?

Reply:
I'm sure it should use it from this, but that is the code as it was before.  I don't want to tweak this right now due to the number of issues already being changed.